components/Badge/badge.css

```diff
@@ -0,0 +1,43 @@
+.txt-color {
+  color: var(--secondary);
+}
+.badge {
+  position: relative;
+  font-size: 1.6rem;
+  display: inline-block;
+}
+.badge__count {
+  padding: 0.3rem 0.45rem;
+  border-radius: 50%;
+  position: absolute;
+  background-color: var(--secondary);
+  color: var(--white);
+  top: -8px;
+  right: -16px;
+  font-size: 0.6rem;
+}
+.badge__status {
```

Comment:
Status badge doesn't seem to intersect with the avatar. Can adjust the status badge to intersect which may make it look better.

Reply:
@DevNirah7 Bro I am not getting it. Do you mean the badges are not properly placed?